tests/test_common_blob.py

```diff
@@ -180,6 +180,19 @@ def test_make_blob_url_with_sas(self):
                          + '.blob.core.windows.net/vhds/my.vhd?sas')
 
     @record
+    def test_make_blob_url_with_snapshot(self):
```

Comment:
Please also add a test case for the scenario in which both sas token and snapshot exist.